vhdl-ts-mode.el

```diff
@@ -627,6 +631,10 @@ Matches if point is at a punctuation/operator char, somehow as a fallback."
   (let ((gen-node (vhdl-ts--node-has-parent-recursive node "\\(for\\|if\\)_generate_statement")))
     (if gen-node
         (treesit-node-start gen-node)
+      (treesit-node-start (treesit-node-parent parent))))
+  (let ((blk-node (vhdl-ts--node-has-parent-recursive node "block_statement")))
+    (if blk-node
+        (treesit-node-start blk-node)
```

Comment:
In theory this anchor should not change as it would also indent concurrent statements of a generate inside a block. 

I guess indentation could be better implemented inside `vhdl-ts--indent-rules` in this case.

Reply:
It exactly should indent concurrent statements of a generate inside a block.